### ISSUE TITLE
Droping AllenNLP from requirements

### ISSUE THREAD

**Issue by tednaseri** [asahi417/tner#38]
Hi @asahi417, I have an issue with AllenNLP as a requirement. 
Indeed, AllenNLP brings some version-conflicts with other requirements of my project. In this case, it is preferable to drop Allen-NLP from the requirements since I have set `CRF=False` during fine-tuning. 

Do you have any idea about that?

Thank you so much

**Comment by finiteautomata** [comment]
Indeed. AllenNLP is now deprecated, and it also comes with several dependencies that interfere with other projects (such as `datasets` or `wandb`). 

Perhaps we can switch to other `CRF` implementation, such as [pytorch-crf](https://pytorch-crf.readthedocs.io/en/stable/).
